### DISCUSSION TITLE
[PL] Funkcja wybierania kolumn w szablonach dynamicznych przy generowaniu PDF na demie gitstable

### DISCUSSION THREAD

**Discussion by hahyes** [YetiForceCompany/YetiForceCRM#14258] [General]
Dobry,

zauważyłem nową funkcję na demie gitstable, która pozwala na dynamiczny szablon PDF z wyborem kolumn:

![image](https://user-images.githubusercontent.com/37104952/108373160-db03f180-71ff-11eb-8137-d15af53aa838.png)

Nie mogę znaleźć tej funkcjonalności w YetiForce 6.1 z Service Packiem v7. Myślałem, że demo gitstable odzwierciedla aktualnego YetiForce 6.1 v7, ale widzę dzisiaj naprawdę dużo zmian. Kiedy można oczekiwać wprowadzenia powyższej funkcjonalności? Bardzo by pomogła przy wdrażaniu się do Yeti. 

I jeszcze, czy jest konkretny branch, który jest odzwierciedlany na demie gitstable? Branch stable ma ostatnie zmiany 3 miesiące temu, więc jak rozumiem, to nie jest to.

**Comment by mariuszkrzaczkowski** [comment]
Aby funkcja była dostępna musisz dodać w szablonie PDF następujący kod:
`$(custom : DynamicInventoryColumnsTable)$`

https://gitstable.yetiforce.com/index.php?module=PDF&parent=Settings&view=Edit&record=36

![image](https://user-images.githubusercontent.com/10198654/108465768-e9472180-7282-11eb-90fb-2423de3677b7.png)

**Reply by hahyes** [reply]
> Aby funkcja była dostępna musisz dodać w szablonie PDF następujący kod:
> `$(custom : DynamicInventoryColumnsTable)$`
> 
> https://gitstable.yetiforce.com/index.php?module=PDF&parent=Settings&view=Edit&record=36
> 
> ![image](https://user-images.githubusercontent.com/10198654/108465768-e9472180-7282-11eb-90fb-2423de3677b7.png)

Niestety, po wklejeniu tej linijki, wybór kolumn się nie pojawia. Po zaimportowaniu szablonu z gitstable również nic.

![image](https://user-images.githubusercontent.com/37104952/108968730-71f9ff00-7681-11eb-92d6-1568c1bc16fd.png)

![image](https://user-images.githubusercontent.com/37104952/108968792-876f2900-7681-11eb-9006-87acafa0a253.png)


Może mam jakąś nieprawidłową wersję? To na pewno 6.1 z Service Packiem v7. Z pliku version.php:
```
<?php

return [
	'appVersion' => '6.1.0',
	'patchVersion' => '2020.12.01',
	'lib_roundcube' => '0.1.5'
];

```

![image](https://user-images.githubusercontent.com/37104952/108968417-0a43b400-7681-11eb-93cf-12b26be9c179.png)

Edit: Teraz widzę, że na gitstable jest wersja Service Pack v5.
